common/lib/dependabot/clients/azure.rb

```diff
@@ -27,10 +35,11 @@ def self.for_source(source:, credentials:)
       # Client #
       ##########
 
-      def initialize(source, credentials)
+      def initialize(source, credentials, max_retries: 3)
         @source = source
         @credentials = credentials
         @auth_header = auth_header_for(credentials&.fetch("token", nil))
+        @max_retries = max_retries || 3
```

Comment:
This is redundant as `max_retries` already has a default argument of 3, but I noticed the other retryable clients follow the same pattern so feel free to leave it in for now

Reply:
Thanks!